exchanges/order/order_test.go

```diff
@@ -297,6 +297,32 @@ func TestTitle(t *testing.T) {
 	require.Equal(t, "Limit", orderType.Title())
 }
 
+var orderComparisonList = []struct {
```

Comment:
This can be merged into TestOrderIs, enhanced with t.Run + t.Parallel with subtests since it isn't used anywhere else